### PR TITLE
Core: increment version

### PR DESCRIPTION
**Pull request by Berserker66**
## What is this fixing or adding?
increment version

### CODE REVIEW

**Review by black-sliver — APPROVED** [review]
Not sure when we want to merge it, but change is fine

File: Utils.py [hunk 1]
```diff
@@ -46,7 +46,7 @@ def as_simple_string(self) -> str:
         return ".".join(str(item) for item in self)
 
 
-__version__ = "0.4.6"
+__version__ = "0.4.7"
```
Comment (nicholassaylor) [inline Utils.py:49]:
```suggestion
__version__ = "0.5.0"
```

**Review by nicholassaylor — CHANGES_REQUESTED** [review]
Reminder to change the version number because of #1933

**Comment by ThePhar** [comment]
> Reminder to change the version number because of #1933

Version is already 0.5.0 as of #1933, so I'll just close this PR.

**Comment by Berserker66** [comment]
Darn, you took my technically by the rules shotgun away.

**Pull request closed without merging**